ci-operator/config/openshift-eng/agent-qe-infra/tools/generate_cron_bm_abi.py

```
@@ -0,0 +1,197 @@
+    # Pre-scan files to identify valid jobs and aggregate weights
+    for dir_str in yaml_dirs:
+        dir_path = Path(dir_str)
+        if not dir_path.exists():
+            continue
+        valid_paths.append(dir_path)
+
+        for yaml_file in dir_path.rglob("*.yaml"):
+            try:
+                content = yaml_file.read_text()
+            except Exception:
+                continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)